### PR TITLE
With this addition the worker-operator would know if the gateway tunnel is down or not

### PR DESCRIPTION
**Pull request by kon3m**
Signed-off-by: kon3m <aakash.dokka@aveshasystems.com>

### CODE REVIEW

File: pkg/sidecar/sidecarpb/gw_sidecar.proto [hunk 1]
```diff
@@ -44,6 +44,11 @@ enum ClassType {
     TBF = 1;
 }
 
+enum TunnelStatusType {
+    GW_TUNNEL_STATE_UP = 0;
+    GW_TUNNEL_STATE_DOWN = 1;
+	GW_TUNNEL_STATE_UNKNOWN = 2;
```
Comment (rahulsawra98) [inline pkg/sidecar/sidecarpb/gw_sidecar.proto:50]: when would be using GW_TUNNEL_STATE_UNKNOWN ?

Comment (kon3m) [reply pkg/sidecar/sidecarpb/gw_sidecar.proto:50]: Not using it, will remove

**Comment by rahulsawra98** [comment]
@kon3m why are there too many vendor changes?

**Comment by kon3m** [comment]
@rahulsawra98 i have rebased it with master

**Comment by rahulsawra98** [comment]
> @rahulsawra98 i have rebased it with master

looks like you updated all te

**Review by rahulsawra98 — APPROVED** [review]

**Pull request merged**